### PR TITLE
Remove steps that delete code sign.md

### PR DESCRIPTION
**Pull request by anthonydresser**
These steps were causing the windows build to fail. I looks at our other pipelines and found these steps were failing silently, so I'm just removing them.

### CODE REVIEW

**Comment by anthonydresser** [comment]
@kburtram Those files no longer exist (I guess maybe a signing tool version upgrade). If look at the steps in a current run, you can see and error in the log each time about the filing not existing.

**Comment by anthonydresser** [comment]
I'll double check the outputs though to make sure they aren't there.

**Comment by kburtram** [comment]
@anthonydresser makes sense.  Thanks!

File: build/azure-pipelines/darwin/sql-product-build-darwin.yml [hunk 1]
```diff
@@ -86,7 +86,7 @@ steps:
           {
             "keyCode": "CP-401337-Apple",
             "operationSetCode": "MacAppDeveloperSign",
-            "parameters": [],
+            "parameters": [],,
```
Comment (Charles-Gagnon) [inline build/azure-pipelines/darwin/sql-product-build-darwin.yml]: Is this extra , intended?

File: build/azure-pipelines/win32/sql-product-build-win32.yml [hunk 2]
```diff
@@ -159,27 +159,20 @@ steps:
...
-            }
-                ],
-            "toolName": "signtool.exe",
-            "toolVersion": "6.2.9304.0"
+              {
+                "parameterName": "VerifyAll",
+                "parameterValue": "/all"
+              }
+            ],
+          "toolName": "sign",
```
Comment (Charles-Gagnon) [inline build/azure-pipelines/win32/sql-product-build-win32.yml]: Why is this the only one you updated?

Comment (anthonydresser) [reply build/azure-pipelines/win32/sql-product-build-win32.yml]: I'm testing if this fixes it.

**Pull request merged**